assets/static/js/table.js

```diff
@@ -0,0 +1,12 @@
+document.addEventListener("DOMContentLoaded", function () {
+  document
+    .querySelectorAll("table.display-table")
```

Comment:
we should change this selector to `datatables`, instead of `display-table`. (and change the reference elsewhere as well)

That way the class semantically describes its use better.